.vscode/settings.json

```diff
@@ -5,5 +5,6 @@
         "**/.lsp": true,
         "**/.shadow-cljs": true,
         "**/node_modules": true
-    }
+    },
```

Comment:
Why is this included ?

Reply:
My bad, should be on gitingnore and I didn't realize